docs/setup/settings.asciidoc

```diff
@@ -22,14 +22,13 @@ to this Kibana instance.
 To send *no* client-side headers, set this value to [] (an empty list).
 `elasticsearch.requestTimeout:`:: *Default: 30000* Time in milliseconds to wait for responses from the back end or
 Elasticsearch. This value must be a positive integer.
-`elasticsearch.shardTimeout:`:: *Default: 0* Time in milliseconds for Elasticsearch to wait for responses from shards. Set
-to 0 to disable.
+`elasticsearch.shardTimeout:`:: *Default: 0* Time in milliseconds for Elasticsearch to wait for responses from shards. Set to 0 to disable.
 `elasticsearch.ssl.certificate:` and `elasticsearch.ssl.key:`:: Optional settings that provide the paths to the PEM-format SSL
-certificate and key files. These files validate that your Elasticsearch backend uses the same key files.
+certificate and key files. These files validate that your Elasticsearch backend uses the same key files. `elasticsearch.ssl.certificate` and `elasticsearch.ssl.key` are required when `xpack.ssl.verification_mode` in Elasticsearch is set to either `certificate` or `full`.
```

Comment:
I'd suggest dropping the statement about "These files validate that your Elasticsearch backend uses the same key files." because it's unclear, perhaps something like:

> These files are used to verify the identity of Kibana to Elasticsearch and are required when `xpack.ssl.verification_mode` in Elasticsearch is set to either `certificate` or `full`

Reply:
That sounds a lot better.